package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "aws-sdk": "^2.439.0",
     "axios": "^0.19.0",
-    "bcrypt": "^3.0.6",
+    "bcrypt": "^5.1.0",
```

Comment:
nice! Thank you for pulling this in, it'll close this super old PR from dependabot too 🙌 
#134